### ISSUE TITLE
Video oral history with additional audio-only derivative

### ISSUE THREAD

**Issue by jronallo** [IIIF/iiif-av#64]
# Description

I have video oral histories. I would like to let a client know that an audio-only derivative is available that could be played or downloaded instead of the talking head video.

## Variation(s)

Are there other cases where the audio content of a video is more important or primary compared to the video content? I could see video of radio shows being similar.

## Proposed Solutions

This could be listed as another choice along with any versions of the video or broken out into a separate choice or annotation. What's the best pattern for making it clear that the audio is equivalent to what is found in the video?

## Additional Background

This is related to work I'm doing to process video oral histories: https://github.com/jronallo/abrizer/issues/4

**Comment by zimeon** [comment]
Nice use case for that seems to be a pretty close match for the pattern in https://github.com/IIIF/iiif-av/blob/master/source/api/av/examples/9.json . We'll probably need to put a bit of thought to enumerating the possible user experiences: both audios play, one auto-muted but offered as option etc.

**Comment by jronallo** [comment]
Yes, in my case the separate audio file should not be played at the same time as the video which will include its own audio. Also it would be nice to say that this isn't some other audio but the audio that is extracted from the video. There's an equivalence of content there that I'm not seeing in example 9. That example seems to be saying that those should both play at the same time, but the separate audio could be a different audio track than the video has.

**Comment by azaroth42** [comment]
You would also want `viewingHint` of `none`, per [11b](https://github.com/IIIF/iiif-av/blob/master/source/api/av/examples/11b.json) to say that the audio should not be played by default.

**Comment by counterintuit** [comment]
Perhaps I'm off the mark but as one behavior I can imagine a video as the master clock with multiple sound tracks available to audition that are also clocked to the video and selectable by use of "radio buttons" or "solo buttons". Conversely, an audio stream could be the master with multiple video streams selectable. The original "time-of-day" is one form of timecode synchronization used among multiple recording devices--as opposed to some arbitrarily chosen master timecode value. Multiple streams and their relationships in time are a huge part of AV production and could have historical implications. Apologies if this is addressed elsewhere.
